IPython/core/crashhandler.py

```diff
@@ -53,6 +53,16 @@
 If you want to do it now, the following command will work (under Unix):
 mail -s '{app_name} Crash Report' {contact_email} < {crash_report_fname}
 
+Please also include in your email:
+- information about your operating system: is it Linux, macOS, Windows, other,
+  and which version, for example Ubuntu 16.04, macOS 10.13.2, Windows 10 Pro,
+  is it 32-bit or 64-bit, ...
+- information on how you installed {app_name}: from binary, compiled from source,
```

Comment:
'compiled from source' doesn't really apply to pure Python projects like IPython. I'd give examples like "using pip or conda, from GitHub, or as part of a Docker container."